app/javascript/components/pages/JobsAuthenticate.jsx

```diff
@@ -1,36 +1,49 @@
 import React, { useState } from 'react';
+import { useCookies } from 'react-cookie';
 import SharedLayout from 'components/layout/SharedLayout';
 import Logo from 'components/icons/Logo';
 import Button from 'components/Button';
+import { postJobsAuthenticate } from '../../datasources';
 
 import 'stylesheets/page';
 import 'stylesheets/jobs_authenticate';
 
 const JobsAuthenticate = () => {
     const [password, setPassword] = useState('');
+    const [hasError, setHasError] = useState(false);
+    // eslint-disable-next-line no-unused-vars
+    const [_, setCookie] = useCookies(['token']);
+
+    const handleLogin = async (e) => {
+        e.preventDefault();
+
+        try {
+            const data = await postJobsAuthenticate(password);
+            setHasError(false);
+            setCookie('token', data.token);
+            window.location.href = '/jobs';
+        } catch (error) {
+            setHasError(true);
+            setPassword('');
```

Comment:
If the password is incorrect, it will set `hasError` to true, which renders a little banner saying the password is invalid!